mkdocs.yml

```
@@ -9,20 +11,27 @@ extra_javascript:
 nav:
   - Home: index.md
   - Build:
-    - Build on Linux: build/linux-build.md
+    - Build with Docker: build/docker-build.md
+    - Build with Linux: build/linux-build.md
```

I'm not a fan of "Build with Linux", it sounds unnatural, but I don't have any idea on how to fix this. Maybe we'll leave it as it was? Or we can do something like this:
```suggestion
    - Build with Docker: build/docker-build.md
    - Build on Linux: build/linux-build.md
```
Or something like this
```suggestion
    - Docker: build/docker-build.md
    - Debian-based distros / Linux: build/linux-build.md
```